dataeng/resources/enrollment-validation-events.sh

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
+
+END_DATE=$(date +%Y-%m-%d -d "$TO_DATE")
+START_DATE=$(date +%Y-%m-%d -d "$FROM_DATE")
+INTERVAL=$START_DATE-$END_DATE
+
+if [ -z "$NUM_REDUCE_TASKS" ]; then
+    NUM_REDUCE_TASKS=$(( $NUM_TASK_CAPACITY + 1 ))
+fi
+env|sort
+
+${WORKSPACE}/analytics-configuration/automation/run-automated-task.sh \
+  EnrollmentValidationWorkflow --local-scheduler \
+  --output-root $OUTPUT_ROOT/$INTERVAL \
+  --interval $INTERVAL \
+  --credentials $CREDENTIALS \
+  --n-reduce-tasks $NUM_REDUCE_TASKS
+
+pip install awscli
+aws s3 ls $OUTPUT_ROOT/$INTERVAL/ | grep "synthetic_enroll"  || true
```

Comment:
What does this line do?  Is it somehow making sure there are synthesized enrollment files present?   If not, will it skip the code below?

Reply:
From what I know of double pipe is that it will run ```true``` only if the previous command(grep in this case) fails. The code below would still be executed.

Reply:
We also have a post build task that checks for synthetic_enroll.log in console output. For edge.

Reply:
Oh, of course.   *That's* what this line is for -- just to get output into the console log.   Sorry to be slow.   It might be worth a comment on that line so I remember the next time I ask.

Reply:
I've removed the post build task that grepped for 'synthetic_enroll.log' for edge. Does it make sense to remove this as well ?

Reply:
Removed.